todo_app/data/trello_items.py

```diff
@@ -0,0 +1,35 @@
+import os, requests
+from todo_app.Item import Item
+
+key = os.getenv("TRELLO_API_KEY")
+token = os.getenv("TRELLO_API_TOKEN")
+board = os.getenv("TRELLO_BOARD_ID")
+headers = {"Accept" : "application/json"}
+
+def get_items() -> list:
+    url = f"https://api.trello.com/1/boards/{board}/lists?key={key}&token={token}&cards=open"
+    response = requests.request("GET", url, headers=headers).json()
```

Comment:
Very minor point, but `requests` has another way of passing in query parameters:

```
url = f"https://api.trello.com/1/boards/{board}/lists"
params = { 'key': key, 'token': token, 'cards': 'open' }
response = requests.request('GET', url, headers=headers, params=params)
```

This is possibly neater when you have many parameters and makes it a bit easier to do things like sharing some between your various functions.
```
def get_auth_params():
    return { 'key': key, 'token': token }
```

Side note: having your key and token defined once at the top of the file will work for now but will fail when we have tests loading environment variables next module. So you might want to move the `key = os...` etc lines into functions. They will need to read the environment variable during the rest of the code execution, instead of when the file is first imported.